### PR TITLE
Refactor Service::NameValidator#valid_name?

### PR DESCRIPTION
**Pull request by vpereira**
Refactor `Service::NameValidator#valid_name?` method

### CODE REVIEW

File: src/api/app/models/service/name_validator.rb [hunk 1]
```diff
@@ -15,11 +15,15 @@ def valid?
     def valid_name?
       return false unless name.is_a?(String)
```
Comment (vpereira) [inline src/api/app/models/service/name_validator.rb:16]: maybe we should invert both lines? before we check if it's `name.blank? || name.size > 200` and then we check the class?

Comment (danidoni) [reply src/api/app/models/service/name_validator.rb:16]: I would replace this type check with a `respond_to?(:blank?)`

Comment (vpereira) [reply src/api/app/models/service/name_validator.rb:16]: > I would replace this type check with a `respond_to?(:blank?)`

I'm not sure, why?

```
[1] pry(main)> nil.blank?
=> true
```

Comment (danidoni) [reply src/api/app/models/service/name_validator.rb:16]: I'm talking about the `unless name.is_a?(String)`.
We use `duck typing` we don't really care if its really a string or not...
Do you agree?

Comment (vpereira) [reply src/api/app/models/service/name_validator.rb:16]: but everything answers to blank no?

```
[2] pry(main)> 1.respond_to?(:blank?)
=> true
```

Comment (vpereira) [reply src/api/app/models/service/name_validator.rb:16]: IMO @danidoni they check if it's string to avoid any XML injection attack.

Comment (danidoni) [reply src/api/app/models/service/name_validator.rb]: That makes sense

**Review by danidoni — APPROVED** [review]

**Pull request merged**